src/test/java/org/opentripplanner/middleware/triptracker/TripSurveySenderJobTest.java

```diff
@@ -180,11 +180,11 @@ void canSelectMostDeviatedJourneyUsingDeviatedPoints() {
     @Test
     void canSelectDeviatedJourneyWithDeviatedPointsThreshold() {
         TrackedJourney journey1 = new TrackedJourney();
-        journey1.longestConsecutiveDeviatedPoints = 6;
+        journey1.longestConsecutiveDeviatedPoints = 3;
```

Comment:
What was the purpose of changing the points here?

Reply:
Because the window was reduced from 60 to 30 minutes, the test was failing. I half the points so they would be within the new 30 min threshold.